src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Properties/AbstractProjectConfigurationProperties.cs

```diff
@@ -149,7 +150,25 @@ public bool RunCodeAnalysis
         public bool DebugSymbols { get => throw new System.NotImplementedException(); set => throw new System.NotImplementedException(); }
         public bool DefineDebug { get => throw new System.NotImplementedException(); set => throw new System.NotImplementedException(); }
         public bool DefineTrace { get => throw new System.NotImplementedException(); set => throw new System.NotImplementedException(); }
-        public string DefineConstants { get => throw new System.NotImplementedException(); set => throw new System.NotImplementedException(); }
+        public string DefineConstants
```

Comment:
is this used anywhere? if not, you can revert :)

Reply:
there are also a couple unused usings that are failing build https://dev.azure.com/dnceng-public/public/_build/results?buildId=28178&view=logs&j=343adee7-d814-546f-dc3f-19ccb8aa9260&t=8dea7d69-d7ce-5872-c9d8-dd4fee7f95a7&l=9991